enhancements/proposals/1665-metadata/README.md

```diff
@@ -0,0 +1,122 @@
+# Support for ScyllaDB Alternator
+
+## Summary
+
+This proposal aims to introduce basic support for handling metadata in managed objects by ScyllaClusters.
+It aims to make minimal changes to satisfy the use cases presented so far.
+
+## Motivation
+
+ScyllaCluster users need to specify labels and annotations for various reasons including external integrations or object discovery / tracking.
+
+### Goals
+
+- Inherit ScyllaCluster labels and annotation into managed objects (aka default labels)
+- Allow specifying distinct labels and annotations for node services
+- Allow specifying distinct labels and annotations for pods
+- Allow specifying distinct labels and annotations for ingresses
+
+### Non-Goals
+
+- Allow unique metadata for every single object we create 
+
+## Proposal
+
+I'd like to use the existing ScyllaCluster.metadata (labels and annotations) and inherit them in every object we create,
+unless there is an override.
+In the case when using an override, the ScyllaCluster.metadata won't be used.
+
+I propose to extend the ScyllaCluster API to allow overriding metadata for pods, node services and ingresses.
+
+In case there is a conflict with managed metadata, the managed metadata will win.
+
+These changes are purely intended for metadata and shall not change the selectors used.
+
+### User Stories
+
+#### DataDog metrics
+As a user I want to configure [DataDog metrics](https://docs.datadoghq.com/containers/kubernetes/prometheus/?tab=kubernetesadv2#setup) for certain pods.
+This is done by adding `ad.datadoghq.com/<CONTAINER_IDENTIFIER>.checks` to Pod's annotations.
+
+#### Third Party Service and Ingress options
```

Comment:
Isn't this already covered with `.spec.exposeOptions.cql.ingress.annotations` and `.spec.exposeOptions.nodeService.annotations`?